<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.1.1 (Python 3.12.8 on win32)" generated="2025-03-25T13:36:00.891124" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\RedBox 016\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2025-03-25T13:36:00.934021" elapsed="5.984695"/>
</kw>
<status status="PASS" start="2025-03-25T13:36:00.933024" elapsed="5.986690"/>
</test>
<status status="PASS" start="2025-03-25T13:36:00.892126" elapsed="6.027588"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2025-03-25T13:36:00.882945" level="WARN">Error in file 'c:\Users\RedBox 016\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
